nat/traversal/pinger.go

```diff
@@ -124,11 +124,20 @@ func (p *Pinger) Start() {
 			log.Info(prefix, "stop pinger called")
 			return
 		case pingParams := <-p.pingTarget:
-			go p.pingTargetConsumer(pingParams)
+			if isPunchingRequired(pingParams) {
+				go p.pingTargetConsumer(pingParams)
+			}
 		}
 	}
 }
 
+func isPunchingRequired(params *Params) bool {
+	if params.ConsumerPort > 0 {
```

Comment:
Just return enough here:
```suggestion
	return params.ConsumerPort > 0
```